### DISCUSSION TITLE
Copilot X and OpenAI Data Sharing Question

### DISCUSSION THREAD

**Discussion by rojan-rijal** [community/community#60629] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, 

When using some of the technical previews such as the nightly build of Copilot, we noticed that the chat feature such as asking it to explain a code, convert code etc will reach out to an endpoint in GitHub. From the response and the API call made to GitHub it seems GitHub then proxies the data to OpenAI. What is the data sharing policy on this? Is this proxied to Azure OpenAI (on-premise version hosted by GitHub) or is it sending to something like public conversational API that OpenAI provides? 

Thanks,

**Comment by davecheney** [comment, marked as answer]
GitHub Copilot's models are hosted on Azure Cognitive Services, not OpenAI.
